content/en/network_performance_monitoring/dns_monitoring.md

```diff
@@ -0,0 +1,92 @@
+---
+title: DNS Monitoring
+kind: documentation
+description: Diagnose and debug DNS server issues  
+aliases:
+    - /network_performance_monitoring/network_table
+further_reading:
+    - link: 'https://www.datadoghq.com/blog/network-performance-monitoring'
+      tag: 'Blog'
+      text: 'Network Performance Monitoring'
+    - link: 'https://www.datadoghq.com/blog/monitor-coredns-with-datadog/'
+      tag: 'Blog'
+      text: 'Monitor CoreDNS with Datadog'
+    - link: '/network_performance_monitoring/network_page'
+      tag: 'Documentation'
+      text: 'Explore network data between each source and destination.'
+    - link: 'https://www.datadoghq.com/blog/dns-resolution-datadog/'
+      tag: 'Blog'
+      text: 'Use DNS resolution to monitor cloud and external endpoints'
+---
+
+{{< img src="network_performance_monitoring/dns_default.png" alt="DNS Monitoring" >}}
+
+DNS Monitoring provides an overview of DNS server performance to help you identify server-side and client-side DNS issues. By collecting and displaying flow-level DNS metrics, this page can be used to identify:
+
+* Which pods or services are making DNS requests, and to which servers
+* Which endpoints are making the most requests, or making requests at the highest rate
+* When a DNS server’s response time to requests has gradually or suddenly increased
+* Which DNS servers have a high error rate, and what kind of errors they they are emitting
+
+## Setup
+
+DNS Monitoring metrics are collected automatically by the system probe. Once installed, a ‘DNS’ tab will be accessible in the Network Performance Monitoring product by default - no extra steps are necessary.
+
+## Queries
+
+You can use the source and destination search bars at the top of the page to query for dependencies between a client (_source_), which makes the DNS request, and a DNS server (_destination_), which responds to the DNS request. The destination port is automatically scoped to DNS port 53 so that all resulting dependencies match this (client → DNS server) format. 
```

Comment:
```suggestion
Use the source and destination search bars at the top of the page to query for dependencies between a client (_source_), which makes the DNS request, and a DNS server (_destination_), which responds to the DNS request. The destination port is automatically scoped to DNS port 53 so that all resulting dependencies match this (client → DNS server) format. 
```